src/uk/gov/hmcts/contino/YarnBuilder.groovy

```diff
@@ -308,6 +308,12 @@ EOF
     return status
   }
 
+  private yarnDepreciationCheck() {
+     if (!isYarnV2OrNewer()){
+       WarningCollector.addPipelineWarning("packageManager required", "Please upgrade to Yarn V3 by adding packageManager to package.json. For examples see: https://github.com/hmcts/cnp-plum-frontend/blob/2bfb6eac580c0965e331731496619c0b5920d596/package.json#L47 and https://github.com/hmcts/dtsse-ardoq-adapter", LocalDate.of(2023, 03, 10 ))
```

Comment:
better if you can send a pull request suggestion for upgrading or pointing to the slack announcement  https://moj.enterprise.slack.com/files/T1L0WSW9F/F04784SLAJC?origin_team=T1L0WSW9F
```suggestion
       WarningCollector.addPipelineWarning("old_yarn_version", "Please upgrade to Yarn V3 by adding packageManager to package.json. For examples see: https://github.com/hmcts/cnp-plum-frontend/blob/2bfb6eac580c0965e331731496619c0b5920d596/package.json#L47 and https://github.com/hmcts/dtsse-ardoq-adapter", LocalDate.of(2023, 03, 10 ))
```